src/main/docker/run-pct.sh

```diff
@@ -157,7 +157,7 @@ mkdir -p "${PCT_OUTPUT_DIR}"
 ###
 # Determine if we test the plugin against another JDK
 ###
-TEST_JDK_HOME=${TEST_JAVA_ARGS:-"/usr/lib/jvm/java-${JDK_VERSION:-8}-openjdk-amd64"}
+TEST_JDK_HOME=${TEST_JAVA_ARGS:-"/usr/local/openjdk-${JDK_VERSION:-8}"}
```

Comment:
AFAICT not going to work for JDK 11 as installed in https://github.com/jenkinsci/plugin-compat-tester/blob/1f0ced625866895896f822e660e5d1012adc659f/Dockerfile#L53

Reply:
```
Cannot run program "/usr/local/openjdk-11/bin/java": error=2, No such file or directory
```

Reply:
Yep, updated in https://github.com/jenkinsci/plugin-compat-tester/pull/277/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R53 to make it happen :-)